version/mkversion/mkversion.go

```diff
@@ -187,7 +187,8 @@ func tailscaleModuleHash(modBs []byte) (string, error) {
 		if i := strings.LastIndexByte(req.Mod.Version, '-'); i != -1 {
 			return req.Mod.Version[i+1:], nil
 		}
-		return "", fmt.Errorf("couldn't parse git hash from tailscale.com version %q", req.Mod.Version)
+		// If there are no dashes, the version is a tag.
```

Comment:
update comment above too? it says `git hash`

Reply:
oh even the func says `tailscaleModuleHash` maybe `tailscaleModuleHashOrTag`

Reply:
Fixed, and also renamed variables and functions to say "ref" instead of "hash" for this value.